src/pq/conninfo.cr

```diff
@@ -171,7 +171,13 @@ module PQ
     end
 
     private def current_user_name
-      System::User.find_by(id: LibC.getuid.to_s).username
+      {% if flag?(:windows) %}
+        # NOTE: actually getting the current username on windows would be better
+        #       https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getusernamew
+        "crystal"
```

Comment:
Instead of using `crystal` would `postgres` be a good alternative? Generally that username will exist. That's what we used as a fallback for Avram

https://github.com/luckyframework/avram/blob/b04ba2d4c130d6ddaf98069bfdedbf8a2b7b935a/src/avram.cr#L35

Reply:
Yeah that makes sense!